routing/index_graph_starter_joints.cpp

```diff
@@ -76,13 +76,12 @@ m2::PointD const & IndexGraphStarterJoints::GetPoint(JointSegment const & jointS
 
 std::vector<Segment> IndexGraphStarterJoints::ReconstructJoint(JointSegment const & joint)
 {
-  // Есть невидимые joint, которые стоят перед стартовыми сегментами, в случае
-  // если они не фейковые, такие мы пропускаем, потому что это просто абстракция
-  // для общности алгоритма.
+  // We have invisible JointSegments, which are come from start to start or end to end.
+  // They need just for generic algorithm work. So we skip such objects.
   if (IsInvisible(joint))
     return {};
 
-  // В случае фейковой вершины, мы возращаем ее предпостроенный путь.
+  // In case of fake vertex we return its prebuild path.
```

Comment:
`// In case of a fake vertex we return its prebuilt path.`

Reply:
done